src/V3AstNodes.h

```diff
@@ -7107,6 +7107,86 @@ class AstPowUS : public AstNodeBiop {
     virtual int instrCount() const { return widthInstrs() * instrCountMul() * 10; }
     virtual bool signedFlavor() const { return true; }
 };
+class AstPreInc : public AstNodeBiop {
```

Comment:
Suggest make AstPreAdd, AstPostAdd, then pass it a Const 0x1 or -1 (which is expressed as all ones) "new AstConst(pinp->fileline(), AstConst::StringToParse(), "'1")"

Then don't need the Sub flavors and also all set to support "+= N"  and "-= N".